plugins/inventory/xen_orchestra.py

```diff
@@ -22,9 +22,9 @@
         - inventory_cache
     options:
         plugin:
-            description: The name of this plugin, it should always be set to C(community.general.xen_orchestra) for this plugin to recognize it as its own.
+            description: The name of this plugin, it should always be set to C(xen_orchestra) for this plugin to recognize it as its own.
             required: yes
-            choices: ['community.general.xen_orchestra']
+            choices: ['xen_orchestra']
```

Comment:
Can you change this back to `community.general.xen_orchestra`? Thanks.

Reply:
Sorry, I was testing a bit of code and I pushed this by accident